scheduler/gform_backend.py

```diff
@@ -123,83 +127,68 @@ def parse_times(time: str) -> float:
     return float(time[0]) + (float(time[1]) / 60)
 
 
-def get_riders(responses: list, days_enabled: list, dues_payers: set) -> list:
-    """Creates a list of riders from the responses.
-
+def get_days_and_locations(start_col: int, response: int,
+                           days_enabled: list) -> list:
+    """
```

Comment:
I spent about 5-10 minutes hunting down why this function is useful and what it does, and I am still a bit confused. From my understanding, when parsing the response data this function will help create a list of DayInfo objects that represent the days and times that someone entered into the form. I think this function just needs a few more comments to clarify its intent. These are the questions I would like to see answered (I didn't want to answer them wrong and add the comments myself, sorry)

1. Summary of what this function does. Should be inserted after the docstring quotes end
1. Typical Usage of this function
1. Clarification of where this data comes from (the sheet right?). I like the diagram in the comments, but I am confused about what it is portraying. More comments about that are needed. When I looked in the response data, it seemed like each member response is one row, yet this drawing depicts data in columns. The distinction between columns and rows here confused me

Reply:
Used lines 223 and 239. See commit [b9cd619](https://github.com/nzsilverman/climbing_carpools/pull/41/commits/b9cd61966b5f71da1d990c55dd0514370dd2a257)